pico/tools/src/main/java/io/helidon/pico/tools/ModuleInfoDescriptor.java

```diff
@@ -44,6 +44,7 @@
  *
  * @see java.lang.module.ModuleDescriptor
  */
+@SuppressWarnings("unused")
```

Comment:
This warning should not be suppressed in sources. Idea supports suppressing this warning by annotations - so if a type is annotated with `@Builder`, it will never be marked as unused